Nos/Views/ExpandingTextFieldAndSubmitButton.swift

```diff
@@ -24,35 +24,41 @@ struct ExpandingTextFieldAndSubmitButton: View {
     
     var body: some View {
         HStack {
-            NoteTextEditor(text: $reply, placeholder: placeholder, focus: focus)
+            NoteTextEditor(text: $reply, placeholder: placeholder, focus: $isFocused)
                 .frame(maxHeight: 270)
                 .background(Color.appBg)
                 .cornerRadius(17.5)
             if showPostButton {
                 Button(
                     action: {
                         disabled = true
-                        focus.wrappedValue = false
+                        isFocused = false
                         Task {
                             await action()
-                            reply = EditableNoteText()
+                            reply = EditableNoteText() // Ensure appropriate initialization
                             disabled = false
                         }
                     },
                     label: {
-                        Localized.post.view
+                        Localized.post.view // Ensure 'Localized' is defined elsewhere
                     }
                 )
                 .disabled(disabled)
             }
         }
-        .onChange(of: focus.wrappedValue) { bool in
-            showPostButton = bool
+        .onChange(of: reply) { newText in
```

Comment:
I think this block is the one that actually fixed the bug. Can you try reverting all the other changes?

Reply:
I did it.